code/game/mecha/combat/gygax.dm

```diff
@@ -31,7 +31,6 @@
 	max_equip = 4
 
 /obj/mecha/combat/gygax/dark/loaded/Initialize()
-	. = ..()

```

Comment:
It's pretty vital for Initialize() to call parent. Don't do this unless you totally know what the heck is up.
Instead, Add a new `auto_mecha_tracking` var (name it however you want) to `/obj/mech/combat` of value FALSE, then set it TRUE for the base Durand, Gygax, H.O.N.K. and Phazon typepaths, then make sure the dark Gygax and dark H.O.N.K. and possibly other special sub-typepaths have it FALSE...